srcpkgs/direnv/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'direnv'
 pkgname=direnv
-version=2.23.1
+version=2.24.0
 revision=1
 build_style=go
 go_import_path=github.com/direnv/direnv
 short_desc="Environment switcher for the shell"
-maintainer="Diogo Leal <diogo@diogoleal.com>"
+maintainer="Sora Morimoto <sora@morimoto.io>"
```

Comment:
Have you talked to @diogoleal about taking over?

Reply:
No.

Reply:
​If it's possible to add more than one maintainer, that would be ideal for me.

Reply:
Since the current maintainer hasn't been active in Void for a while, it should be ok in this case. Just note that usually it's better to send them an email or at least mention them in the PR.

Reply:
I got it. Thanks for your explanation.

Indeed. ​For example, a template in the Alpine Linux source package collection may have multiple maintainers, but I haven't found yet with Void Linux.